packages/firebase_admob/lib/firebase_admob.dart

```diff
@@ -130,8 +132,16 @@ abstract class MobileAd {
   ///
   /// The [listener] will be notified when the ad has finished loading or fails
   /// to do so. An ad that fails to load will not be shown.
-  Future<bool> show() {
-    return _channel.invokeMethod("showAd", <String, dynamic>{'id': id});
+  Future<bool> show({int anchorOffset, AnchorType anchorType}) {
+    final Map<String, dynamic> params = <String, dynamic>{'id': id};
+    if (anchorOffset != null) {
+      params['anchorOffset'] = anchorOffset;
```

Comment:
Could we make the `anchorType` default to `AnchorType.bottom` and the `anchorOffset` default to `0.0` ---and then avoid dealing with null/nil in the platform-specific code?